includes/class-wc-stripe-webhook-handler.php

```
@@ -1391,6 +1391,133 @@ public function process_account_updated( $notification ) {
 	}
 
+	/**
+	 * Processes the checkout session completed event.
+	 *
+	 * @param object $notification The notification from Stripe
+	 * @return void
+	 */
+	public function process_checkout_session( object $notification ): void {
+		$checkout_session = $notification->data->object;
+
+		$order = WC_Stripe_Helper::get_order_by_checkout_session_id( $checkout_session->id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'process_checkout_session'. (undefined)

(StaticAccess)